metis-indexing/src/test/java/eu/europeana/indexing/tiers/view/MediaResourceTechnicalMetadataBuilderTest.java

```diff
@@ -44,6 +44,8 @@ void objectCreationTest() {
     assertThrows(NullPointerException.class, mediaResourceTechnicalMetadataBuilder::build);
     final MediaTier mediaTier = MediaTier.T0;
     mediaResourceTechnicalMetadataBuilder.setMediaTier(mediaTier);
+    final MediaTier mediaTierBeforeLicense = MediaTier.T1;
```

Comment:
This is not tested later down on the assertions?
Perhaps other tests are similar.

Reply:
You are right, I forgot to add the extra assertion. I just fixed that and I'll check other tests